MRG Deprecates 'normalize' in LinearRegression (_base.py) (#17743)
scikit-learn/scikit-learn: doc/whats_new/v0.24.rst

  :pr:`16289` by :user:`Masashi Kishimoto <kishimoto-banana>` and
  :user:`Olivier Grisel <ogrisel>`.

:mod:`sklearn.linear_model`
...........................

- |API|: The parameter ``normalize`` of :class:`linear_model.LinearRegression` is deprecated and will be removed

>   in 0.26. The use of :class:`~sklearn.pipeline.Pipeline` with :class:`~sklearn.preprocessing.StandardScaler` is recommended instead.

Reviewer: careful about line length please ;)

  :pr:`16289` by :user:`Masashi Kishimoto <kishimoto-banana>` and
  :user:`Olivier Grisel <ogrisel>`.

:mod:`sklearn.linear_model`
...........................

> - |API|: The parameter ``normalize`` is deprecated in 0.24 and will be removed

Reviewer: ```suggestion
- |API|: The parameter ``normalize`` of :class:`linear_model.LinearRegression` is deprecated and will be removed
```

>   in 0.26. The use of pipeline with StandardScaler is recommended instead.

Reviewer: ```suggestion
  in 0.26. The use of :class:`~sklearn.pipeline.Pipeline` with :class:`~sklearn.preprocessing.StandardScaler` is recommended instead.
```

  ``normalize`` parameter did not take any effect if ``fit_intercept`` was set
  to False and therefore was deemed confusing.

>   :class:`linear_model.LinearRegression`

Reviewer: ```suggestion

```
